packages/features/ee/workflows/lib/reminders/scheduleMandatoryReminder.ts

```diff
@@ -37,6 +38,8 @@ export async function scheduleMandatoryReminder(
         const filteredAttendees =
           evt.attendees?.filter((attendee) => attendee.email.includes("@gmail.com")) || [];
 
+        if (isPlatformNoEmail) return;
```

Comment:
We could move this to top of the try or even outside of it at the very top of function because if `isPlatformNoEmail` then we still do checks and filter and then return nothing anyway.